easybuild/easyblocks/t/tensorflow.py

```diff
@@ -427,3 +442,5 @@ def sanity_check_step(self):
                 mnist_py = os.path.join(self.start_dir, 'tensorflow', 'examples', 'tutorials', 'mnist', mnist_py)
                 cmd = "%s %s --data_dir %s" % (self.python_cmd, mnist_py, tmpdir)
                 run_cmd(cmd, log_all=True, simple=True, log_ok=True)
+
+        return res
```

Comment:
Should this res be additive with the mnist test result?

Reply:
As it is now, if the mnist test fails, it will raise an error straight away rather than postponing it until the result of the sanity check for extensions is reported.

That's probably good enough, since `TensorFlow` will typically be the last extension being installed.

Changing this in a way that works both for `TensorFlow` installed as an extension and stand-alone is a bit messy...

Reply:
Ok, good enough